app/controllers/stats_controller.rb

```diff
@@ -21,6 +21,16 @@ def range
                   .joins('INNER JOIN users ON users.uid = rusers.id')
                   .where('users.status = 1')
                   .count
+    @questions = Node.questions
+                   .select(%i[created type status])
+                   .where(type: 'note', status: 1, created: @start.to_i..@end.to_i)
+                   .count   
+   @answers = Answer.where(type: 'note', status: 1, created: @start.to_i..@end.to_i)
```

Comment:
Ah, here, Answers don't have `type` or `status`, I believe. And we may have to measure `created_at: @start..@end` instead of converting to integers here. Unfortunately `Node` and `Comment` don't have standard timestamps since they were from a legacy database, but `Answer` does -- that /should/ work if I'm not mistaken. If you can make this change, and the `Gemfile.lock` reversion, we can likely merge this! Thank you so much!